environments/contrail/install_vrouter_kmod.yaml

```diff
@@ -213,7 +213,7 @@ resources:
                 fi
               fi
             fi
-            if [[ `echo $role |awk -F"-" '{print $2}'` == "novacompute" || `hostname |awk -F"-" '{print $2}'` == "contrailtsn" ]]; then 
+            if [[ `echo $role |awk -F"-" '{print $2}'` == "compute" || `hostname |awk -F"-" '{print $2}'` == "contrailtsn" ]]; then
```

Comment:
I'm sorry @alexey-mr but we are talking about Role and not hostname here. And the your file, the role is compute : 

`https://github.com/Juniper/contrail-tripleo-heat-templates/blob/master/environments/contrail/roles_data.yaml#L100-L102`